### ISSUE TITLE
LPOP returns false instead of null when key is not exist

### ISSUE THREAD

**Issue by phoenixgao** [phpredis/phpredis#675]
Phpredis lpop returns false instead of null when key is not exist, while redis returns nil

shouldn't return null?

**Comment by michael-grunder** [comment]
Yes this is probably correct.  There are a few cases like this in phpredis actually.  Let me take a look and see if I can solve it in a generic way for all such instances.  I'll need to be careful though, as people are certainly relying on this in some applications.
